From: Michael Biebl <biebl@debian.org>
Date: Tue, 17 Aug 2021 20:42:31 +0200
Subject: Disable integration/rhbz1928860.at test

It requires NetworkManager to be built with OVS suppport which is not
the case in Debian (yet).
---
 src/tests/integration/networkmanager.at | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/tests/integration/networkmanager.at b/src/tests/integration/networkmanager.at
index 0b20adc..3b059e8 100644
--- a/src/tests/integration/networkmanager.at
+++ b/src/tests/integration/networkmanager.at
@@ -1,3 +1,3 @@
 AT_BANNER([NetworkManager (FIREWALL_BACKEND)])
 m4_include([integration/rhbz1773809.at])
-m4_include([integration/rhbz1928860.at])
+dnl m4_include([integration/rhbz1928860.at])
